From 820d2ef3596189041da43d1139da8689336bb918 Mon Sep 17 00:00:00 2001
From: Changqing Li <changqing.li@windriver.com>
Date: Mon, 26 Nov 2018 11:19:45 +0800
Subject: [PATCH] Subject: [PATCH] Always use devmapper for kpartx

Do not try to compute the LIBDM_API_COOKIE make variable
from host information when cross-compiling.

Upstream-Status: Inappropriate [embedded specific]

Signed-off-by: Aws Ismail <aws.ismail@windriver.com>
Signed-off-by: Joe Slater <joe.slater@windriver.com>

Rebase to 0.7.1
Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>

Rebase to 0.7.7
Signed-off-by: Changqing Li <changqing.li@windriver.com>

Rebase to 0.7.9
Signed-off-by: Changqing Li <changqing.li@windriver.com>
[OP: Rebase to 0.9.3]
Signed-off-by: Ovidiu Panait <ovidiu.panait@windriver.com>
---
 kpartx/Makefile | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/kpartx/Makefile b/kpartx/Makefile
index 742d3bcd..d886483d 100644
--- a/kpartx/Makefile
+++ b/kpartx/Makefile
@@ -9,9 +9,7 @@ LDFLAGS += $(BIN_LDFLAGS)
 
 LIBDEPS += -ldevmapper
 
-ifneq ($(call check_func,dm_task_set_cookie,$(DEVMAPPER_INCDIR)/libdevmapper.h),0)
-	CPPFLAGS += -DLIBDM_API_COOKIE
-endif
+CPPFLAGS += -DLIBDM_API_COOKIE
 
 OBJS = bsd.o dos.o kpartx.o solaris.o unixware.o dasd.o sun.o \
 	gpt.o mac.o ps3.o crc32.o lopart.o xstrncpy.o devmapper.o
-- 
2.38.1

